File: Ejercicio/Java/JSON/TextoJSON.java
import org.json.JSONObject;

public class TextoJSON {
    public static void main(String[] args) {
        String texto = "{\"nombre\":\"Carlos\",\"edad\":30,\"profesion\":\"Ingeniero\"}";
        JSONObject objeto = new JSONObject(texto);

        System.out.println("Nombre: " + objeto.getString("nombre"));
        System.out.println("Edad: " + objeto.getInt("edad"));
    }
}

File: Ejercicio/GO/JSON/TextoJSON.go
package main

import (
	"encoding/json"
	"fmt"
)

func main() {
	producto := map[string]interface{}{
		"id":     1,
		"nombre": "Portátil",
		"precio": 2500.5,
	}

	jsonTexto, _ := json.Marshal(producto)
	fmt.Println("Objeto como texto JSON:", string(jsonTexto))
}
